Makefile

```diff
@@ -129,7 +117,7 @@ test: examples test-stable
 	LIBTOCK_PLATFORM=hifive1 cargo clippy $(EXCLUDE_STD) \
 		--target=riscv32imac-unknown-none-elf --workspace
 	MIRIFLAGS="-Zmiri-strict-provenance -Zmiri-symbolic-alignment-check" \
-		cargo miri test $(EXCLUDE_MIRI) --workspace
+		cargo +nightly miri test $(EXCLUDE_MIRI) --workspace
```

Comment:
We need to pin to a particular nightly release, otherwise this will break randomly as new nightly toolchains come out.